Code/GraphMol/Descriptors/Lipinski.cpp

```diff
@@ -469,5 +472,30 @@ unsigned int calcNumBridgeheadAtoms(const ROMol &mol,
   return atoms->size();
 }
 
+const std::string NumAtomStereoCentersVersion = "1.0.0";
+unsigned int numAtomStereoCenters(const ROMol &mol) {
```

Comment:
assignStereochemistry takes a non-const mol.  I'm not sure I can call it here without a lot of refactoring.  Any suggestions?  return NaN?  Throw an exception?  Certainly log an error...